desktop/core/src/desktop/js/apps/storageBrowser/StorageBrowserPage/StorageBrowserTabContents/StorageBrowserTabContent.tsx

```diff
@@ -62,6 +62,10 @@ const StorageBrowserTabContent: React.FC<StorageBrowserTabContentProps> = ({
   const [pageStats, setPageStats] = useState<PageStats>();
   const [pageSize, setPageSize] = useState<number>();
   const [pageNumber, setPageNumber] = useState<number>(1);
+  const [sortByColumn, setSortByColumn] = useState<string>('');
+  const [sortByDescending, setSortByDescending] = useState<boolean>(false);
```

Comment:
How does this work when you only have one boolean for descending sort but three states: no-sort, descending and ascending? Have a look at this which shows what I'm referring to  https://react.carbondesignsystem.com/?path=/story/components-datatable-selection--with-selection-and-sorting

Unless the API requires that there is always at least one sorted column I think that the user must be able to "unsort" a sorted table and get the original order back.

Reply:
I believe in the current implementation of file browser once the user sorts any column there is no way of having an unsure option unless you refresh the page. This is exactly what happens in this version as well. Initially the sortByColumn is an empty string which is the same as unsort option.